src/oss/confluent.sh

```diff
@@ -1402,7 +1402,9 @@ connect_load_command() {
             connector_config_template "${connector}" "${confluent_conf}/${_retval}" "true"
             parsed_json="${_retval}"
         else
-            die "${connector} is not a predefined connector name.\nUse '${command_name} load ${connector} -d <connector-config-file.[json|properties]' to load the connector's configuration."
```

Comment:
I wonder if a better fix is to have `die` call `printf` instead of `echo`.

Reply:
Considered that (or using `echo -e`). My take was that this is the only error message we use with more than one line and there may be unexpected errors caused either now or in the future by enabling interpretation of backslash escape sequences in our error messages, so a more conservative approach seemed warranted.

Reply:
I'm not too concerned about breaking previous behavior. We'd just have to review how `die` is used so far. Having said that, I agree that this is just one case, so a less general fix is fine.